src/test/js/edgeCases/folder.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
             const freestylePage = browser.page.pipeline()
                 .forUrl(response.value, jobName);
 
+            // FIXME: Talk to Tom, why next lines are not working as expected
             // freestylePage.buildStarted(function () {
```

Comment:
yeah, right but the pipeline() should be renamed to jobUtils or something https://github.com/jenkinsci/blueocean-acceptance-test/blob/master/src/main/js/page_objects/classic_jenkins/pipeline.js I am reusing that. So freestyleJob is as well more explicit for job()